recipes/syft/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "syft" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: syft
+  version: "0.2.5"
+
+source:
+  url: https://files.pythonhosted.org/packages/07/67/ea53ea6cef94f2ff09696c901fe70fca32407a86e627c5ea52738663061f/syft-0.2.5.tar.gz
+  sha256: 155690cf8745348f8d358e697319cec93f53cd0014456cf7f07e9f8ea47c5333
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
+  run:
+    - pip
+    - python
+    - pytest
+    - flask_socketio~=4.2.1
+    - Flask~=1.1.1
+    - lz4~=3.0.2
+    - msgpack~=1.0.0
+    - numpy~=1.18.1
+    - phe~=1.4.0
+    - Pillow~=6.2.2
+    - requests~=2.22.0
+    - scipy~=1.4.1
+    - syft-proto~=0.4.1
+    - tblib~=1.6.0
+    - torchvision~=0.5.0
```

Use different requirements annotation
```suggestion
    - torchvision 0.5.0
```